### ISSUE TITLE
use GeoSPARQL for relations instead of spatial, osspatial

### ISSUE THREAD

**Issue by VladimirAlexiev** [isawnyu/pleiades-rdf#12]
https://github.com/isawnyu/pleiades-rdf#user-content-geovocab. Pleiades uses `spatial:C, osspatial:partiallyOverlaps` for relations.

For similar reasons to #4, I would strongly recommend using GeoSPARQL relations. GeoSPARQL Sec 7 Topology Vocabulary Extension (relation_family) defines 3 families of topological relations: Simple, RCC8, Egenhofer=DE-9IM. Sec 7.5 defines equivalences between these families as appropriate. For the ones Pleiades has:
- contains: geo:sfContains = geo:ehContains+geo:ehCovers = geo:rcc8tppi+geo:rcc8ntppi 
   (RCC8 distinguishes between tangential and non-tangential proper part)
- partially overlaps: geo:sfOverlaps = geo:ehOverlap = geo:rcc8po

For simplicity, use `geo:sfContains, geo:sfOverlaps`

**Comment by VladimirAlexiev** [comment]
@ewg118 do you agree?

**Comment by ewg118** [comment]
Makes sense. I haven't done much with GeoSPARQL directly, only the spatial extension to Jena, which is technically not GeoSPARQL
